packages/components/src/TextArea/_docs/TextArea.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import { LinkTo } from "~storybook/components/LinkTo"
+import * as TextAreaStories from "./TextArea.stories"
+
+<Meta of={TextAreaStories} />
+
+# TextArea
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/TextArea"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { TextArea } from "@kaizen/components"'
+/>
+
+## Overview
+
+Usually part of compositions such as <LinkTo pageId="components-text-area-field">TextAreaField</LinkTo>
```

Comment:
Suggestion: we could document that this can be used by itself, BUT they need to link up a label manually.